tests/unit/h/search/parser_test.py

```diff
@@ -127,26 +127,24 @@ def test_parse_with_odd_quotes_combinations(query_in, query_out):
     assert parser.parse(query_in) == query_out
 
 
-@given(st.text())
-@settings(max_examples=30)
-def test_parse_always_return_a_multidict(text):
-    """Given any string input, output should always be a MultiDict."""
```

Comment:
mypy can verify the "always returns a MultiDict" part, provided parsing doesn't throw. I assume the intent here was a fuzz test that `parse` doesn't throw an exception for any string input. We could generate a bunch of random strings here via some other means, but this doesn't seem super high value.